test/behaviour/graql/language/get/BUILD

```diff
@@ -38,7 +38,7 @@ java_test(
         "@graknlabs_behaviour//graql/language:get.feature",
     ],
     classpath_resources = ["//test/conf:logback"],
-    size = "small",
+    size = "medium",
```

Comment:
Why are we increasing the size of all the tests?

Reply:
they all take longer now -- im pretty sure its cuz we open twice the number of databases as before!